.github/workflows/verify.yml

```
@@ -12,10 +12,24 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 12.x
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Cache build
+        uses: actions/cache@v2
+        with:
+          path: .cache/webpack
+          key: ${{ runner.os }}-build-${{ hashFiles('**/package-lock.json') }}
       - name: Install dependencies
         run: npm ci
       - name: Build website
-        run: npm run build
+        run: npm run build:dev
+        env:
+          TEST_ENV: true # Slice icon list
```

Didn't notice this before, I don't feel like this is necessary here so I prefer to remove it - that's one less place to change it if this is changed in the future.

```suggestion
        run: npm run build:dev
```

---

I think it would be better to use prod build, as we always "test" to make sure it is available in a production environment, not a test environment. Using prod build also doesn't seem to result in less debugging information, so it won't affect local debug adversely. I'll revert related changes.